bin/sem-apply

```diff
@@ -8,24 +8,33 @@
 #  sem-apply --host <database host> --user <db user> --name <db name>
 #
 # == Examples
-#  sem-apply --url postgresql://postgres@localhost/sample
+#  sem-apply --url postgresql://postgres@localhost:port/sample
 #  sem-apply --host localhost --user web --name test
 #
+require 'tempfile'
 
 load File.join(File.dirname(__FILE__), 'sem-config')
 
 args = SchemaEvolutionManager::Args.from_stdin(:optional => %w(url host port name user dry_run))
-
 db = SchemaEvolutionManager::Db.from_args(args)
-db.bootstrap!
-
 dry_run = args.dry_run.nil? ? false : args.dry_run
+pgpass = args.pgpass.nil? ? false : args.pgpass
+pgpassfile = Tempfile.new('pgpass')
+util = SchemaEvolutionManager::ApplyUtil.new(db, pgpassfile, :dry_run => dry_run)
 
-util = SchemaEvolutionManager::ApplyUtil.new(db, :dry_run => dry_run)
-
-puts "Upgrading schema for #{db.url}"
 
 begin
+
+  if pgpass
+    puts "please enter your password (or press enter to continue)"
+    password = SchemaEvolutionManager::Ask.for_password()
+    if !(password.nil? || password.empty?)
```

Comment:
I think if the password is empty, we should re-prompt until we get a password. ie. if I specify --pgpass, make me enter a password. Otherwise I can remove the --pgpass parameter